README.md

```diff
@@ -1,23 +1,34 @@
 # Calico Networking for rkt
 
-## Building the plugin
-To build the calico-kubernetes plugin, clone this repository and run `make`.  This will build the binary, as well as run the unit tests.  To just build the binary, with no tests, run `make binary`.  To only run the unit tests, simply run `make ut`.
+`calico-rkt` offers basic Calico networking for rkt deployments.
 
 ## Using rkt Plugins
-The [documentation](https://github.com/coreos/rkt/blob/master/Documentation/networking.md) for Network Plugins in rkt will walk you through the basics of setting up networking in rkt.
+
+The CoreOS [documentation](https://github.com/coreos/rkt/blob/master/Documentation/networking.md) for Network Plugins will walk you through the basics of setting up networking in rkt.
 
 ## Requirements
+
 * A working [etcd](https://github.com/coreos/etcd) service
-* Though Calico is capable of networking rkt containers, our core software is distributed and deployed in a [docker container](https://github.com/projectcalico/calico-docker/blob/master/docs/getting-started/default-networking/Demonstration.md). For now, you will need to run calico in docker before starting your rkt contianers. This can be easily done by downloading the [calicoctl binary](https://github.com/projectcalico/calico-docker/releases) and running the following command: `calicoctl node --ip=$IP`
+* A build of `calicoctl` after [projectcalico/calico-docker@10460cc405](https://github.com/projectcalico/calico-docker/commit/10460cc405f5aa4bc9ccb1fcaf8760088ae1ebf9)
```

Comment:
There should be a link to the calico-docker releases page with a minimum release.  Otherwise there isn't any way for the user to know where to find calicoctl.